README.md

```diff
@@ -21,7 +21,7 @@ Setup
 
 ### Using Published Plugin
 
-For sbt 0.13.6+ add sbt-assembly as a dependency in `project/assembly.sbt`:
+For sbt 0.13.6+ add sbt-assembly as a dependency in `project/plugins.sbt`:
```

Comment:
This is intentional. See http://jsuereth.com/scala/2013/06/11/effective-sbt.html